design/ceph/object/ceph-rgw-support-multisite-for-external-cluster.md

```diff
@@ -0,0 +1,50 @@
+---
+# Improvements to RGW Multisite configuration
+target-version: release-1.10
+---
+
+# Feature Name
+ Improvements to RGW Multisite configuration
```

Comment:
Could you summarize the gaps in the current multisite config? I'm having difficulty understanding why we need the features suggested in this design.